src/routes/docs/products/functions/executions/+page.markdoc

```diff
@@ -57,6 +57,280 @@ Each execution can have one of the follow status.
 - The function execution was not successful.
 {% /table %}
 
+Function executions can be delayed if you want to trigger them later.
+
+# Delayed executions {% #delayed-executions %}
+
+Delayed executions trigger the function only once at a future date and time. To schedule an execution, navigate to **Your function** > **Executions** > **Execute now** > **Schedule** in the Appwrite Console.
+
+{% only_dark %}
+![Add a SMTP provider](/images/docs/functions/execution/dark/scheduled-execution-function.png)
+{% /only_dark %}
+{% only_light %}
+![Add a SMTP provider](/images/docs/functions/execution/scheduled-execution-function.png)
+{% /only_light %}
+
+You can also schedule your function executions using a [runtime](/docs/products/functions/runtimes#available-runtimes).
```

Comment:
I initially was using SDK here, but I used runtime here because the function runtimes are different from the SDKs, but if I want to use runtime here instead of SDK, I think there could be a better way I could write this.